Makefile

```diff
@@ -74,8 +74,10 @@ grpc:
 $(grpc_services_directory):
 	@echo "grpc service directory $(directory) does not exist, generating services"
 	$(REBAR) get-deps
-	$(MAKE) grpc
+	REBAR_CONFIG="config/grpc_server_gen.config" $(REBAR) grpc gen
```

Comment:
Could this be `make grpc` instead to avoid duplicating the generation steps?

Reply:
I tried that and it fell into a loop.